pkg/ccl/changefeedccl/changefeed_dist_test.go

```diff
@@ -480,6 +481,72 @@ func (rdt *rangeDistributionTester) balancedDistributionUpperBound(numNodes int)
 	return int(math.Ceil((1 + rebalanceThreshold.Get(&rdt.lastNode.ClusterSettings().SV) + 0.1) * 64 / float64(numNodes)))
 }
 
+func TestChangefeedWithDistributionStrategyOptions(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+
+	skip.UnderShort(t)
+	skip.UnderDuress(t)
+	skip.UnderStress(t)
```

Comment:
Also, now that we've changed what the test is doing, maybe we don't need these skips anymore?

Reply:
We briefly discussed this offline but noting for posterity that I'd like us to at least understand why it's failing under stress before skipping it (and maybe adding a comment documenting the reason).